lib/src/showcase.dart

```diff
@@ -250,56 +262,75 @@ class _ShowcaseState extends State<Showcase> with TickerProviderStateMixin {
     Size size,
     Rect rectBound,
     Size screenSize,
-  ) =>
-      Visibility(
-        visible: _showShowCase,
-        maintainAnimation: true,
-        maintainState: true,
-        child: Stack(
-          children: [
-            GestureDetector(
-              onTap: _nextIfAny,
-              child: Container(
-                width: MediaQuery.of(context).size.width,
-                height: MediaQuery.of(context).size.height,
-                child: CustomPaint(
-                  painter: ShapePainter(
-                    opacity: widget.overlayOpacity,
-                    rect: position!.getRect(),
-                    shapeBorder: widget.shapeBorder,
+  ) {
+    var blur = widget.blur == null
+        ? ShowCaseWidget.of(context)!.defaultBlur
+        : widget.blur!;
+
+    // Set blur to 0 if application is running on web and
+    // provided blur is less than 0.
+    blur = kIsWeb && blur < 0 ? 0 : blur;
+
+    return _showShowCase
+        ? Stack(
+            children: [
+              GestureDetector(
+                onTap: _nextIfAny,
+                child: ClipPath(
+                  clipper: RRectClipper(
+                    area: rectBound,
+                    isCircle: widget.shapeBorder == CircleBorder(),
                     radius: widget.radius,
-                    color: widget.overlayColor,
```

Comment:
what is the reason for removing this ?

Reply:
previously we were using custom painter now we are using custom clipper. so it is moved to line no: 293 and 301.